src/booking-list/components/CompleteRideWindow.tsx

```diff
@@ -17,6 +17,8 @@ export const CompleteRideWindow = ({
 }: ICompleteRideWindow) => {
   const navigate = useNavigate();
 
+  console.log('rideCompletionText', rideCompletionText);
```

Comment:
In future PRs, try to remove console.logs and commented parts unless they are absolutely necessary